pkg/aggregator/tagger.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !serverless
+
+package aggregator
+
+import (
+	"github.com/DataDog/datadog-agent/comp/core/tagger"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/collectors"
+	"github.com/DataDog/datadog-agent/pkg/tagset"
+)
+
+func enrichTags(tb tagset.TagsAccumulator, udsOrigin string, clientOrigin string, cardinalityName string) {
```

Comment:
These functions are using global instance of tagger which should be migrated to a injected component. 
For example:
```
func enrichTags(tb tagset.TagsAccumulator, udsOrigin string, clientOrigin string, cardinalityName string, taggerComp tagger.Component) {
	taggerComp.EnrichTags(tb, udsOrigin, clientOrigin, cardinalityName)
}
```
We can get pass in a global tagger instance (by calling GetTaggerInstance()) at this moment and try to remove GetTaggerInstance in the future.